posts/urls.py

```diff
@@ -8,4 +9,5 @@
     url(r'^(?P<pk>\d+)/edit/$', views.PostEditView.as_view(), name='edit'),
     url(r'^(?P<pk>\d+)/repost/$', views.PostRepostView, name='repost'),
     url(r'^(?P<pk>\d+)/favorite/$', views.PostFavoriteView, name='favorite'),
+    url(r'^show/(?P<hashtag_name>\w+)/$', PostsListView.as_view(), name='posts_list'),
```

Comment:
`/posts/hashtags/foobar` (note the **s**) would be my personal choice, as it's the closest to REST.
My reasoning is that you look for posts (not e.g. users), and in the posts, you look for hashtags (not e.g. mentions or authors).
`/hashtags/foobar` would also be ok, but hashtag search should be encapsulated into the posts app IMO.